### PR TITLE
Add contributing guide

### LINKED ISSUE #42: Contributing guide
We should add a `CONTRIBUTING.md` showing how people can contribute to Stryker4s, as well as some common stuff like what kind of issues to look for when you are new to mutation testing or Scala, or how to run Stryker4s on Stryker4s. Also see [Stryker's contributing guide](https://github.com/stryker-mutator/stryker/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by hugo-vrijswijk**
Fixes #42

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+This is the contribution guide for Stryker4s. Great to have you here! Here are a few ways you can help to make this project better.
+
+## Adding a new feature
+
+New features are welcome! Both as ideas or in the form of a pull request.
+
+1. Please [create an issue](https://github.com/stryker-mutator/stryker4s/issues/new) with your idea first or let us know via [Gitter](https://gitter.im/stryker-mutator/stryker4s).
+2. Create a fork on your GitHub account.
+3. When writing your code, please conform the existing coding style. This can be as easy as running `sbt scalafmt` to format your code. Or take a moment to review the [Scala style guide](https://docs.scala-lang.org/style/).
```
Comment (legopiraat) [inline docs/CONTRIBUTING.md]: Or have the latest version of intellij and configure it to use .scalafmt

Comment (hugo-vrijswijk) [reply docs/CONTRIBUTING.md]: Added. The scalafmt documentation isn't wholly up to date anymore as intellij now has scalafmt support built in and doesn't need the plug-in anymore

**Comment by legopiraat** [comment]
Do we want to add a Team section as well like Stryker? If not this is ready to be merged

**Comment by hugo-vrijswijk** [comment]
I'm not sure. People can already contact us on Gitter or whatever. And at what point would we add new members to the team? Just seems like something that would get out of date fairly quickly (at least, that would be nice). People can always look at the git history ;)

**Pull request merged**